### PR TITLE
toolbar: authentication button changed

### PR DESCRIPTION
**Pull request by kkrisstoff**
Make auth button in same style like all buttons for extended toolbar
beta:
beta
My PR:
now

### CODE REVIEW

**Comment by yanas** [comment]
It looks weird to have the authentication separated from the Profile which is now on the top of the extended toolbar.  I think that we should add the login button in the Profile side container and remove the authentication button.

File: interface_config.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ var interfaceConfig = { // eslint-disable-line no-unused-vars
     // the toolbar buttons line is intentionally left in one line, to be able
     // to easily override values or remove them using regex
     MAIN_TOOLBAR_BUTTONS: ['microphone', 'camera', 'desktop', 'invite', 'hangup'], // jshint ignore:line
-    TOOLBAR_BUTTONS: ['profile', 'authentication', 'microphone', 'camera', 'desktop', 'recording', 'security', 'raisehand', 'chat', 'etherpad', 'sharedvideo', 'fullscreen', 'sip', 'dialpad', 'settings', 'hangup', 'filmstrip', 'contacts'], // jshint ignore:line
+    // list of buttons for left toolbar
+    EXTENDED_TOOLBAR_BUTTONS: ['profile', 'contacts', 'chat', 'security', 'etherpad', 'settings', 'raisehand', 'fullscreen',  'filmstrip'], // jshint ignore:line
+    TOOLBAR_BUTTONS: ['profile', 'microphone', 'camera', 'desktop', 'recording', 'security', 'raisehand', 'chat', 'etherpad', 'sharedvideo', 'fullscreen', 'sip', 'dialpad', 'settings', 'hangup', 'filmstrip', 'contacts'], // jshint ignore:line
```
Comment (kkrisstoff) [inline interface_config.js]: I removed "authentication", because now it isn't a button

File: modules/UI/toolbars/Toolbar.js [hunk 2]
```diff
@@ -387,11 +387,9 @@ Toolbar = {
      * @param show <tt>true</tt> to show or <tt>false</tt> to hide
      */
     showAuthenticateButton (show) {
-        if (UIUtil.isButtonEnabled('authentication') && show) {
```
Comment (kkrisstoff) [inline modules/UI/toolbars/Toolbar.js]: I am testing it in `UI.updateAuthInfo` now

File: modules/UI/toolbars/Toolbar.js [hunk 3]
```diff
@@ -459,7 +459,7 @@ Toolbar = {
      * @param show <tt>true</tt> to show
      */
     showLoginButton (show) {
-        if (UIUtil.isButtonEnabled('authentication') && show) {
```
Comment (kkrisstoff) [inline modules/UI/toolbars/Toolbar.js]: it is duplicate functionality

File: modules/UI/toolbars/Toolbar.js [hunk 4]
```diff
@@ -6,7 +6,68 @@ import SideContainerToggler from "../side_pannels/SideContainerToggler";
 let emitter = null;
 let Toolbar;
 
+/**
+ * Opens the invite link dialog.
+ */
+function openLinkDialog () {
```
Comment (yanas) [inline modules/UI/toolbars/Toolbar.js]: This code reverts a change that has been done in the latest invite dialog code. You need to re-merge your code with the latest master and have a look at the changes done by @BeatC on invite dialogs.

File: interface_config.js [hunk 5]
```diff
@@ -20,6 +20,8 @@ var interfaceConfig = { // eslint-disable-line no-unused-vars
     // the toolbar buttons line is intentionally left in one line, to be able
     // to easily override values or remove them using regex
     MAIN_TOOLBAR_BUTTONS: ['microphone', 'camera', 'desktop', 'invite', 'hangup'], // jshint ignore:line
+    // list of buttons for left toolbar
+    EXTENDED_TOOLBAR_BUTTONS: ['profile', 'contacts', 'chat', 'security', 'etherpad', 'settings', 'raisehand', 'fullscreen',  'filmstrip'], // jshint ignore:line
```
Comment (yanas) [inline interface_config.js]: The TOOLBAR_BUTTONS already contains the list of extended toolbar buttons. That's why we have 2 lists MAIN_TOOLBAR_BUTTONS for the top toolbar and TOOLBAR_BUTTONS for the extended toolbar.

File: modules/UI/toolbars/Toolbar.js (hunk 4 above)
Comment (kkrisstoff) [inline modules/UI/toolbars/Toolbar.js]: Yep. If was wrong conflict resolve. I already fixed it.

File: modules/UI/UI.js [hunk 6]
```diff
@@ -1140,11 +1140,12 @@ UI.notifyFocusLeft = function () {
  * @param {string} [login] current login
  */
 UI.updateAuthInfo = function (isAuthEnabled, login) {
+    let showAuth = isAuthEnabled && UIUtil.isAuthenticationEnabled();
     let loggedIn = !!login;
 
-    Toolbar.showAuthenticateButton(isAuthEnabled);
```
Comment (yanas) [inline modules/UI/UI.js]: Why do you get rid of isAuthEnabled and replace it by showAuth?

File: interface_config.js [hunk 7]
```diff
@@ -17,6 +17,10 @@ var interfaceConfig = { // eslint-disable-line no-unused-vars
     GENERATE_ROOMNAMES_ON_WELCOME_PAGE: true,
     APP_NAME: "Jitsi Meet",
     INVITATION_POWERED_BY: true,
+    /**
+     * If we should show authentication block in profile
+     */
+    AUTHENTICATION_ENABLE: true,
```
Comment (yanas) [inline interface_config.js]: Authentication UI should be enabled in the way it was done previously, we cannot replace the current logic with a simple UI property. See my comments below.

File: modules/UI/UI.js [hunk 8]
```diff
@@ -1140,11 +1140,12 @@ UI.notifyFocusLeft = function () {
  * @param {string} [login] current login
  */
 UI.updateAuthInfo = function (isAuthEnabled, login) {
+    let showAuth = isAuthEnabled && UIUtil.isAuthenticationEnabled();
```
Comment (yanas) [inline modules/UI/UI.js]: Ok, now I see that you take isAuthEnabled into account actually, but also use the UI property. Why do we need the UI property then?

**Pull request merged**